axon/deep_net.go

```diff
@@ -298,9 +298,9 @@ func (nt *Network) AddPTPredLayer(ptMaint, ct, thal *Layer, ptToPredPrjn, ctToPr
 	ptPred.PlaceBehind(ptMaint, space)
 	nt.ConnectCtxtToCT(ptMaint, ptPred, ptToPredPrjn).SetClass("PTtoPred")
 	nt.ConnectLayers(ct, ptPred, ctToPredPrjn, emer.PrjnType(ForwardPrjn)).SetClass("CTtoPred")
-	toThal, fmThal := nt.BidirConnectLayers(ptMaint, thal, toThalPrjn)
-	toThal.SetClass("PredToThal")
-	fmThal.SetClass("ThalToPred")
+	// toThal, fmThal := nt.BidirConnectLayers(ptPred, thal, toThalPrjn)
```

Comment:
do we want this to stay commented out?

Reply:
good point -- just discovered this was being done late last night -- now removed.

Reply:
and added comment why